homeassistant/components/melcloud/config_flow.py

```diff
@@ -56,9 +72,39 @@ async def _create_client(
                 )
         except ClientResponseError as err:
             if err.status in (HTTPStatus.UNAUTHORIZED, HTTPStatus.FORBIDDEN):
+                if self.context["source"] == config_entries.SOURCE_IMPORT:
```

Comment:
Can't you create a wrapper around async_abort at this point where you check if the source is import, if so, raise an issue with `deprecated_yaml_import_issue_{reason}`?

Reply:
I did something along that way to simplify and reuse the code a bit now